safenode/src/bin/cli/files.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use safenode::{
+    client::{Client, Files},
+    protocol::address::ChunkAddress,
+};
+
+use bytes::Bytes;
+use clap::Parser;
+use eyre::Result;
+use std::{fs, path::PathBuf};
+use tracing::info;
+use walkdir::WalkDir;
+use xor_name::XorName;
+
+#[derive(Parser, Debug)]
+pub enum FilesCmds {
+    Upload {
+        /// The location of the files to upload.
+        #[clap(name = "files-path")]
+        files_path: PathBuf,
+    },
+    Download {
+        /// The location of the file names stored
+        /// when uploading files.
+        #[clap(name = "file-names-path")]
```

Comment:
I would clarify the name of this argument and how it makes the command behave. Are you going to be supplying many different file paths? Or should it be the location of one directory, and all the files in that directory would be downloaded? It would be worth elaborating the doc comments there to describe the behaviour. If appropriate I would also consider renaming it to just `path`, since again, it is localised to a `files download` command.

Reply:
Okay, yeah.

So this will load a new file that we created, where we stored all file addresses that we uploaded.

So it will load that file, deserialize into a vec and iterate through it to download the previously uploaded files.

Reply:
File names path is something different than download path.

So it's where the file names are found (they are serialized into a file that was created in the upload dir of files that were uploaded, all this is temporary, it's just one iteration on top of what was even more temporary before it..).

So, leaving that description out I feel will be confusing. I.e. we are not specifying a download path here.